store/kv/kv.go

```diff
@@ -60,6 +60,7 @@ type KeyValueStore interface {
 	CreateTable(ctx context.Context, name []byte) error
 	DropTable(ctx context.Context, name []byte) error
 	GetInternalDatabase() (interface{}, error) // TODO: CDC remove workaround
+	TableSize(ctx context.Context, name []byte) (int64, error)
```

Comment:
So Calling Table for anything may be confusing, we are using Table terminology in the code for encoded `namespace, database, collection`, should we just call it `KeyRangeSize` here?

Reply:
Currently there is a table concept used in every KV method.
I think it's easier to understand that keys are scoped by table and it'll be easier to move to other KV backend just in case.
But we can think and change it in later diffs, it would be a big change.